vllm/v1/core/kv_cache_manager.py

```diff
@@ -89,6 +101,16 @@ def __init__(
         self.num_cached_block: Dict[str, int] = defaultdict(int)
         self.prefix_cache_stats = PrefixCacheStats()
 
+        # The following swap maps are accumulated over a scheduling step.
+        # Then they are "flushed" as part of the scheduler output.
+        # GPU block ID -> CPU block ID
+        self.step_d2h_swap_map: Dict[int, int] = {}
```

Comment:
Could you please do an abstract to support `Offload to Disk` in the future? If we did this abstraction, the data structure can be [ [src_device, dst_device] -> swap_map[src_block_id -> dst_block_id] ]. Any throughs?